### PR TITLE
Updated Launch code

### PR DESCRIPTION
**Pull request by notzippy**
Added output to error stack, so terminal errors are displayed
Ficed c.Verbose, it was changed to an array which causes issues launching
Removed . notation from doing anything special. This was already replaced with the -p CLI option
Added documentaiton on adding the package name
Started watcher with force refresh.

### CODE REVIEW

**Comment by pedromorgan** [comment]
just curious why the `.vscode/` directory is versioned ?

**Comment by notzippy** [comment]
> 

Just makes it easier for someone else to debug the code

File: harness/build.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ import (
 
 var importErrorPattern = regexp.MustCompile("cannot find package \"([^\"]+)\"")
 var importErrorPattern2 = regexp.MustCompile("no required module provides package ([^;]+)+")
+var addPackagePattern = regexp.MustCompile(`to add:\n\tgo get (.*)\n`)
```
Comment (brendensoares) [inline harness/build.go:29]: `go get` is [deprecated](https://go.dev/doc/go-get-install-deprecation)

File: harness/build.go [hunk 2]
```diff
@@ -214,6 +215,10 @@ func Build(c *model.CommandConfig, paths *model.RevelContainer) (_ *App, err err
 		if matches == nil {
 			matches = importErrorPattern2.FindAllStringSubmatch(stOutput, -1)
 		}
+		if matches == nil {
```
Comment (brendensoares) [inline harness/build.go:218]: why are we duplicating the  `if` condition above? Why not add to it?

**Pull request merged**